src/index.ts

```diff
@@ -127,7 +180,8 @@ function getEmit(
     // Emit Javascript
     const output = instance.languageService.getEmitOutput(filePath);
 
-    loader.clearDependencies();
+    // Remove original dependency on resourcePath and replace with rawFilePath
```

Comment:
To be honest I think the GitHub UI is making things more confusing than I would hope 😄 

I'm still a little confused and at least part of that is down to my incomplete knowledge of the webpack API.  (Just for history, ts-loader was originally written by @jbrantly and I'm the main caretaker at present.)  I'll lay out my confusion and you can set me straight.  

As I read it, `removeDependencies` does the following:
- it gathers up existing `contextDependencies` and `dependencies` from webpack.
- clears dependencies
- re-adds the previously gathered dependencies **unless** they are the `resourcePath` - could you clarify what the intent of this is please?  I can see what's happening but I'm not so clear as to what you want to **not** be added as a dependency and what you do.